scripts/spack/packages/axom/scr_examples_gtest.patch

```diff
@@ -0,0 +1,50 @@
+diff --git a/src/axom/sidre/examples/spio/CMakeLists.txt b/src/axom/sidre/examples/spio/CMakeLists.txt
```

Comment:
Spack's CI found a failure in our SPIO examples file.  Basically gtest was being added regardless of ENABLE_TESTS/AXOM_ENABLE_TESTS.  So `axom~tests+examples` woud fail.  This patches the last two releases in Spack.